### ISSUE TITLE
Rainbow text not scrolling through

### ISSUE THREAD

**Issue by joeellis** [joeellis/barcamp2012#34]
@bhelx - I changed up some of your code so that I could use the scrolling rainbow text everywhere (mainly set it to use getElementsByClassName instead of getElementsById).  However, it's only working on the very last span.  I figure I have some "pass by reference" issue somewhere and I'm just overlooking something.

Any idea what might be the cause of it?

https://github.com/joeellis/barcamp2012/blob/master/public/js/rainbow.js

**Comment by joeellis** [comment]
Yes, it's definitely something minor / stupid caused by me.  If you spend more than 5 minutes looking at it and don't see the problem, then feel free to forget about it and close out the issue.  I'll consider errors as first class features on this site.

**Comment by bhelx** [comment]
I know the problem. It's a scoping issue. Just verified in console. I'll make a PR at noon.
